### PR TITLE
DX-2188 Update Samples for StartGather

### PR DESCRIPTION
**Pull request by ckoegel**

### CODE REVIEW

**Review by hamermike — APPROVED** [review]

File: site/docs/voice/bxml/startGather.mdx [hunk 1]
```diff
@@ -69,30 +69,68 @@ There is no text content available to be set for the `<StartGather>` verb.
...
 </Response>
 ```
 
 </TabItem>
 <TabItem value="java">
 
-:::note
-Coming Soon
-:::
+```
```
Comment (abaldwin-Bandwidth) [inline site/docs/voice/bxml/startGather.mdx:79]: add Java tag

File: site/docs/voice/bxml/startGather.mdx [hunk 2]
```diff
@@ -114,16 +152,37 @@ console.log(response.toBxml());
...
+print(response.to_bxml())
+```
 
 </TabItem>
 <TabItem value="php">
 
-:::note
-Coming Soon
-:::
+```
```
Comment (abaldwin-Bandwidth) [inline site/docs/voice/bxml/startGather.mdx:174]: add PHP tag

File: site/docs/voice/bxml/startGather.mdx (hunk 1 above)
Comment (ckoegel) [inline site/docs/voice/bxml/startGather.mdx:79]:
```suggestion
```java
```

File: site/docs/voice/bxml/startGather.mdx (hunk 2 above)
Comment (ckoegel) [inline site/docs/voice/bxml/startGather.mdx:174]:
```suggestion
```php
```

**Review by hamermike — APPROVED** [review]

**Pull request merged**